### PR TITLE
Stop double counting event on the firefox/lockwise page (Fixes #7941)

### LINKED ISSUE #7941: Double counting events on lockwise page
## Description

In https://github.com/mozilla/bedrock/blob/d09bd3bdce9a778c951fce74fe9d58866629fa81/bedrock/firefox/templates/firefox/lockwise/lockwise.html#L55, we see the following:

```
 <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button"  data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
```

I believe `data-link-type` causes an event to fire, for an iOS (browser) download. And `data-cta-type` causes an event to fire. We should not fire both events.

We should use the spec here for the data attributes on links on this page:

https://github.com/mozilla/bedrock/issues/7583#issuecomment-536762745

Specifically:

* remove data-link-type and data-download-os from the lockwise app store links

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removed GA events on app store badges that are double-counting as Firefox mobile downloads.

## Issue / Bugzilla link
#7941

## Testing
URLs:

- http://localhost:8000/en-US/firefox/lockwise/
- http://localhost:8000/en-US/exp/firefox/lockwise/
- http://localhost:8000/de/exp/firefox/lockwise/

Things to test:

- [ ] Badges **should have** `data-cta-text`, `data-cta-type`, and `data-cta-position` attributes.
- [ ] Badges **should not have** `data-link-type` or `data-download-os` attributes.

### CODE REVIEW

**Comment by amychurchwell** [comment]
not pertinent to this PR, but I think it is worth noting that the `GOOGLE_PLAY_LOCKWISE_LINK` value includes `?id=mozilla.lockbox` when the product is lockwise? Should it be `mozilla.lockwise`?

File: bedrock/exp/templates/exp/firefox/lockwise.html [hunk 1]
```diff
@@ -44,10 +44,10 @@
     ) %}
 
       <div class="mobile-download-buttons">
-        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
+        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
           <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="Download on the App Store" width="152" height="45">
         </a>
-        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), extra_data_attributes={'download-location': button_location, 'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
+        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), product='lockwise', extra_data_attributes={'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
```
Comment (amychurchwell) [inline bedrock/exp/templates/exp/firefox/lockwise.html]: not a blocker- but maybe we should consider adding the option to add a param on adjust links to indicate if it is an experiment page.

File: bedrock/exp/templates/exp/firefox/lockwise.de.html [hunk 2]
```diff
@@ -44,10 +44,10 @@
     ) %}
 
       <div class="mobile-download-buttons">
-        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" data-link-type="download" data-download-os="iOS" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
+        <a href="{{ lockwise_adjust_url('ios', 'lockwise-page') }}" class="ios-button" data-cta-text="apple-app-store" data-cta-type="lockwise-app-store" data-cta-position="primary">
           <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="Im App Store downloaden" width="152" height="45">
         </a>
-        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), extra_data_attributes={'download-location': button_location, 'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
+        {{ google_play_button(href=lockwise_adjust_url('android', 'lockwise-page'), product='lockwise', extra_data_attributes={'cta-type': 'lockwise-app-store', 'cta-text': 'google-play-store', 'cta-position': 'primary' } ) }}
```
Comment (amychurchwell) [inline bedrock/exp/templates/exp/firefox/lockwise.de.html]: nit: should `product='Lockwise'` to match `product='Firefox'` and `product='Focus'`?

**Comment by alexgibson** [comment]
> not pertinent to this PR, but I think it is worth noting that the GOOGLE_PLAY_LOCKWISE_LINK value includes ?id=mozilla.lockbox when the product is lockwise? Should it be mozilla.lockwise?

Changing the URL to `mozilla.lockwise` is a 404. I think we need to keep it as-is.

File: bedrock/exp/templates/exp/firefox/lockwise.html (hunk 1 above)
Comment (alexgibson) [reply bedrock/exp/templates/exp/firefox/lockwise.html]: Perhaps, but that's not something we have any real specs for as these URLs are adjust links, and do not do FxA auth. It's not something related to fixing this bug, so I don't think it's worth complicating things here?

File: bedrock/exp/templates/exp/firefox/lockwise.de.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/exp/templates/exp/firefox/lockwise.de.html]: I'll capitalize for consistency 👍

**Comment by alexgibson** [comment]
Thanks for the review, @amychurchwell!

**Pull request merged**